.circleci/config.yml

```diff
@@ -139,6 +143,13 @@ references:
   store_test_logs: &store_test_logs
     store_artifacts:
         path: log/test
+  report_code_coverage: &report_code_coverage
+    run:
+        name: Report Code Coverage
+        command: |
+          export ci_env=`bash <(curl -s https://codecov.io/env)`
+          . install/setup.sh
```

Comment:
Do we need to source the workspace to run codecove?

Reply:
I think so, but I'm not 100% sure. It might be that we have to do it because we don't have all the right exec dependencies listed in our package.xml files.